Examples/Io/Root/include/ActsExamples/Io/Root/RootMeasurementWriter.hpp

```diff
@@ -279,18 +103,20 @@ class RootMeasurementWriter final : public WriterT<MeasurementContainer> {
                      const MeasurementContainer& measurements) override;
 
  private:
+  struct DigitizationTree;
+
   Config m_cfg;
-  std::mutex m_writeMutex;  ///< protect multi-threaded writes
-  TFile* m_outputFile;      ///< the output file
-  Acts::GeometryHierarchyMap<std::unique_ptr<DigitizationTree>>
-      m_outputTrees;  ///< the output trees
-  std::unordered_map<Acts::GeometryIdentifier, const Acts::Surface*>
-      m_dSurfaces;  ///< All surfaces that could carry measurements
+  /// protect multi-threaded writes
+  std::mutex m_writeMutex;
+  /// the output file
+  TFile* m_outputFile;
+  /// the output tree
+  std::unique_ptr<DigitizationTree> m_outputTree;
```

Comment:
This is so you don't have to put it in the header, or any other reason?

Reply:
Mostly because I kept the previous implementation